virt/virt-2-4-release-notes.adoc

```diff
@@ -100,15 +100,16 @@ from being inadvertently started before an upload has completed.
 
 [id="virt-2-4-known-issues"]
 == Known issues
-//https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
-//https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
+//This is work in progress.
 
 * KubeMacPool is disabled in {VirtProductName} {VirtVersion}. This means that a secondary
 interface of a Pod or virtual machine obtains a randomly generated MAC address rather
 than a unique one from a pool. Although rare, randomly assigned MAC addresses can conflict.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1816971[*BZ#1816971*])
+//Merge conflict resolution - adding this text back on July 21: //Bug 1795410 - this known issue was open when I checked on July 17th.
```

Comment:
Please remove this text if it is no longer needed.

Reply:
Removed the commented text on merge conflict resolution.